code/modules/cm_marines/dropship_equipment.dm

```diff
@@ -247,75 +247,75 @@
 	var/deployment_cooldown
 	var/obj/machinery/marine_turret/premade/dropship/deployed_turret
 
-	initialize()
-		if(!deployed_turret)
-			deployed_turret = new(src)
-			deployed_turret.deployment_system = src
-		..()
+/obj/structure/dropship_equipment/sentry_holder/initialize()
+	if(!deployed_turret)
+		deployed_turret = new(src)
+		deployed_turret.deployment_system = src
+	..()
 
-	New()
-		if(!deployed_turret)
-			deployed_turret = new(src)
-			deployed_turret.deployment_system = src
-		..()
+/obj/structure/dropship_equipment/sentry_holder/New()
```

Comment:
Why do some of these use new and some use initialize?

Reply:
Done.